packages/p2p/src/components/modal-manager/modals/order-details-confirm-modal/order-details-confirm-modal.tsx

```diff
@@ -113,4 +106,4 @@ const OrderDetailsConfirmModal = () => {
     );
 };
 
-export default OrderDetailsConfirmModal;
+export default observer(OrderDetailsConfirmModal);
```

Comment:
could you please write observer at the beginning?
`const OrderDetailsConfirmModal = observer(() => {`
and keep it
`export default OrderDetailsConfirmModal;`

Reply:
hm.. i think the other way is better.. and in p2p we follow the other way round.. might be different for different packages.

Reply:
ok.. didn't knew things are different in p2p 😄